aip/general/0151.md

```diff
@@ -129,6 +129,20 @@ metadata message. The errors themselves **must** still be represented with a
 Changing either the `response_type` or `metadata_type` of a long-running operation
 is a breaking change.
 
+## Rationale
+
```

Comment:
I think we need a subsection here - this isn't rationale for the entire API just the `### Validate-only behavior`

Reply:
Done in an extra commit.